code/modules/library/random_books.dm

```diff
@@ -27,7 +27,7 @@
 	icon_state = "random_bookcase"
 	anchored = TRUE
 
-/obj/structure/bookcase/random/New()
+/obj/structure/bookcase/random/Initialize()
```

Comment:
This should actually fix random bookcases. Currently, because they `/New()`, they are created before `world/New()` has progressed, thus trying to spawn books without a database connection existing.